sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/PropagatorConfigurationTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.autoconfigure;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.context.propagation.ContextPropagators;
+import java.util.Collections;
+import org.junit.jupiter.api.Test;
+
+class PropagatorConfigurationTest {
+
+  @Test
+  void defaultPropagators() {
+    ContextPropagators contextPropagators =
+        PropagatorConfiguration.configurePropagators(
+            ConfigProperties.createForTest(Collections.emptyMap()));
+
+    assertThat(contextPropagators.getTextMapPropagator().fields())
+        .containsExactlyInAnyOrder("traceparent", "tracestate", "baggage");
```

Comment:
I could be missing something, but should "tracecontext" be one of the propagators found given your explicitly adding it?

Reply:
these are the fields for the 2 propagators. I don't know of any great way to get at the propagators themselves.